designer/client/designer.tsx

```diff
@@ -97,9 +60,8 @@ export default class Designer extends Component<Props, State> {
     this.setState({ flyoutCount: --currentCount }, callback());
   };
 
-  saveData = async (toUpdate, callback = () => {}) => {
-    const { id } = this.state;
-    const dataJson = await this.designerApi.save(id, toUpdate);
+  save = async (toUpdate, callback = () => {}) => {
+    const dataJson = await this.designerApi.save(this.id, toUpdate);
```

Comment:
unused var (strange pre-commit lint-stage is not helping with this... will investigate ASAP)

Reply:
this one should be ok - I renamed `saveData` to `save` and removed the old method